pytensor/link/pytorch/dispatch/subtensor.py

```diff
@@ -0,0 +1,69 @@
+from pytensor.link.pytorch.dispatch.basic import pytorch_funcify
+from pytensor.tensor.subtensor import (
+    AdvancedIncSubtensor1,
+    AdvancedSubtensor,
+    AdvancedSubtensor1,
+    IncSubtensor,
+    Subtensor,
+    indices_from_subtensor,
+)
+
+
+@pytorch_funcify.register(Subtensor)
+@pytorch_funcify.register(AdvancedSubtensor)
+@pytorch_funcify.register(AdvancedSubtensor1)
+def pytorch_funcify_Subtensor(op, node, **kwargs):
+    idx_list = getattr(op, "idx_list", None)
+
+    def subtensor(x, *ilists):
+        indices = indices_from_subtensor(ilists, idx_list)
+        new_indices = []
+
+        for i in indices:
+            new_start, new_step, new_stop = None, None, None
+            if isinstance(i, slice):
+                if i.start:
+                    new_start = i.start.item()
```

Comment:
I think this is not needed if we don't typify integers and tensor by default. I think they are originally integers?